### ISSUE TITLE
[feature request] file selection box to select ffmpeg.exe additional to download option

### ISSUE THREAD

**Issue by paradonym** [ShareX/ShareX#5356]
I use a portable ShareX on multiple systems with multiple paths.
on one system it's a windows share, on another it's a direct UNC path where it's running from.

everything works, just FFMPEG:
if it's downloaded on the machine with the windows share ShareX seems to memorize FFMPEG.exe's file location in it's config the other machine (without access to that share - accessing it with the UNC path) obviously can't find for example: `I:\Portable\ShareX\Tools\FFmpeg.exe`.

It's better to save ffmpeg.exe's file path to `(current sharex.exe working folder) .\Tools\ffmpeg.exe` instead of it's full path name to make ShareX portable.

I can't re-download ffmpeg on the other machine (because it would then destroy my other portable access) - so I need to have ffmpeg's path made portable or a changed ffmpeg.exe missing dialog changed to a file selection dialog to select an already downloaded ffmpeg.exe

for a workaround: where can I find the config file to change sharex filepath to ffmpeg fom I:\Portable\ShareX\Tools\Tools\FFmpeg.exe to `.\Tools\ffmpeg.exe`? (or to `\\some-share\Portable\ShareX\Tools\ffmpeg.exe` if ShareX can't handle `.\`)

Is FFMPEG accessed differently in the installed variant?

TL;DR
file selection dialog for an already downloaded ffmpeg.exe additionally to an option to download it. Alternatively: change config from full-path to short-path `.\` (from sharex.exe's current working folder)

**Comment by gitgt** [comment]
I support the title "Add file selection box to select (existing) ffmpeg.exe additional to download option".
Because I now have 5 ffmepg, each program adds it's own, which is a waste of space.
